pageserver/src/tenant/mgr.rs

```diff
@@ -247,7 +249,7 @@ pub async fn shutdown_all_tenants() {
     let mut tenants_to_freeze_and_flush = Vec::with_capacity(tenants_to_shut_down.len());
     for (_, tenant) in tenants_to_shut_down {
         // updates tenant state, forbidding new GC and compaction iterations from starting
-        tenant.set_stopping().await;
+        let _ = tenant.set_stopping().await; // TODO handle error
```

Comment:
Might not even require much, but marking TODO.

Reply:
It's fixed in the subsequent commit, already in this branch. Seems like you're reviewing commit by commit

Reply:
Ah, TODO got implemented already.